### ISSUE TITLE
Enhance Welcome Dialog

### ISSUE THREAD

**Issue by Apollon77** [ioBroker/ioBroker.matter#262]
The Welcome Dialog should get some enhancements:

* Below the "Welcome" headline and before the "i" boxes we should add a text:
"Der Matter-Adapter erlaubt die Integration von ioBroker mit allen Matter-Geräten auf dem Markt, und gestattet es ioBroker-Geräte in vielen Matter-kompatiblen Ökosystemen, wie beispielsweise Apple, Google, Amazon oder auch SmartThings, direkt zu nutzen. Die Kommunikation erfolgt hierbei lokal und sicher.

Damit dies alles funktioniert sollten die folgenden Anforderungen beachtet werden:"

* When IPv6 was detected and active then the box could be green and the "i" could be a green checkmark? (if not too much effort)

* Add another info-Box after IP6 to show when System runs inside Docker:
"Da ioBroker in Docker ausgeführt wird muss das Host-Netzwerk genutzt werden, da sonst IPv6 und UDP Routing nicht funktioniert. Mehr Details im  [link]Troubleshooting-Guide[/link]."

* please add another info box with:
"Bei Problemen beim Finden von Geräten oder beim Verknüpfen mit Ökosystemen müssen potentiell weitere IPv6-Einstellungen aiuf dem ioBroker Host vorgenommen werden. Bitte für Details im [link]Troubleshooting-Guide[/link] nachlesen."

**Comment by JackDaniel007** [comment]
wunsch:
das mit **kein ipv6** bei der schnittstelle sollte in einer  signalfarbe (zb. rot) dort stehen, so übersieht man die info schnell (insbesondere beim dunklen hintergrund)

**Comment by Feuer-sturm** [comment]
matter 0.3.5
- 🟢 When IPv6 was detected and active then the box could be green and the "i" could be a green checkmark? (if not too much effort)
- 🟢 Below the "Welcome" headline and before the "i" boxes we should add a text:
- 🟢 please add another info box with: "Bei Problemen beim Finden von Geräten oder beim Verknüpfen mit Ökosystemen müssen potentiell weitere IPv6-Einstellungen aiuf dem ioBroker Host vorgenommen werden. Bitte für Details im [link]Troubleshooting-Guide[/link] nachlesen."
- 🔴 Link of text "[Weitere Einzelheiten finden Sie im Leitfaden zur Fehlerbehebung](https://google.com/)" is wrong and shows to google

Not possible to test:
- Add another info-Box after IP6 to show when System runs inside Docker
- das mit kein ipv6 bei der schnittstelle sollte in einer signalfarbe (zb. rot) dort stehen, so übersieht man die info schnell

**Comment by d-berk** [comment]
matter 0.3.5

🟢Add another info-Box after IP6 to show when System runs inside Docker
🔴 Link of text "[Weitere Einzelheiten finden Sie im Leitfaden zur Fehlerbehebung] is wrong and shows to google.com

![grafik](https://github.com/user-attachments/assets/b13fb454-08d0-473a-9cc9-d650a60698fa)

**Comment by GermanBluefox** [comment]
> 🔴ipv6 **not** .i.O (screenshot 2)

May be other interface has IPv6?

**Comment by d-berk** [comment]
correct, adapter found other Docker-containers that had an ipV6-address.

checked it with a non ipv6-docker-environment and it works as it should.
🟢 no ipv6 detectet

![grafik](https://github.com/user-attachments/assets/21dfda3f-b465-4cdd-ba50-109c3f69c602)

![Screenshot 2025-01-10 165225](https://github.com/user-attachments/assets/b7d08123-f625-42e3-905b-dea742ac2c70)

**Comment by GermanBluefox** [comment]
I changed the behavior of this warning. If some IPv6 was found but the current selected interface does not have IPv6, it will be shown to user the interface selector, so he can pick the valid one.

**Comment by d-berk** [comment]
🟢Link of text "[Weitere Einzelheiten finden Sie im Leitfaden zur Fehlerbehebung] is correct now
